client/my-sites/importer/newsletter/content.tsx

```diff
@@ -96,7 +96,7 @@ export default function Content( {
 					<p>
 						{ createInterpolateElement(
 							__(
-								'Generate a ZIP file of all your Substack posts. On Substack, go to Settings > Exports, click <strong>New export</strong>, and upload the downloaded ZIP file in the next step.'
+								'Generate a ZIP file of all your Substack posts. On Substack, go to Settings > Imports/Exports, click <strong>New export</strong>, and upload the downloaded ZIP file in the next step.'
```

Comment:
When I click on the link to go to the Substack page, I'm seeing Import / Export singular:

<img width="634" alt="Screenshot 2025-05-28 at 10 48 38 AM" src="https://github.com/user-attachments/assets/7414e618-9fd3-4ebc-99ac-6de2f7bb2ff6" />